src/main/java/org/folio/circulation/domain/LoanAction.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.circulation.domain;
+
+public enum LoanAction {
```

Comment:
Thank you for extracting this class, I think it improves type safety and readability

Reply:
Anytime ^^